packages/tupaia-web/src/features/EntitySearch/EntitySearch.tsx

```diff
@@ -33,33 +30,49 @@ const ResultsWrapper = styled.div`
   padding: 0 0.3rem 0.625rem;
   width: calc(100% + 5px);
   border-radius: 8px;
-  max-height: calc(80vh - 12rem);
+  max-height: calc(100vh - 9rem);
   overflow-y: auto;
+
+  @media screen and (max-width: ${MOBILE_BREAKPOINT}) {
+    // Todo: make mobile version of this
+    position: fixed;
+    top: 92px;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    max-height: calc(100vh - 92px);
+  }
 `;
 
 const SearchResults = styled.div`
   padding: 1rem;
   display: flex;
 `;
 
+const isMobile = () => {
+  return window.innerWidth < parseInt(MOBILE_BREAKPOINT, 10);
+};
+
 export const EntitySearch = () => {
   const { projectCode } = useParams();
   const { data: project } = useProject(projectCode!);
   const { data: entity, isLoading } = useEntities(projectCode!, project?.entityCode, {
     enabled: !!project?.entityCode,
   });
   const [searchValue, setSearchValue] = useState('');
-  const [isSearchFocused, setIsSearchFocused] = useState(false);
+  const [isOpen, setIsOpen] = useState(false);
+
+  const onClose = () => {
```

Comment:
the mobile menu is not in scope for this ticket but I wanted to get it working a bit for testing mobile

Reply:
That makes sense. When the time comes to do that, one thing to watch is whether `isMobile` gets called heaps of times or not when it is used like `if(isMobile())` or whether we need to do something to stop it re-rendering the component. I'm not sure if it will be an issue, but I was thinking about it when working on the chart components `isMobile` function

Reply:
Good point - let's do some testing when we do the mobile menu ticket